CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 - Composer script for PHP linting
 
 ### Changed
+- Moved PHP linting to GitHub actions (#12)
```

Comment:
… and also #28

Reply:
I don't see why we need PR + issue ID, esp. since the PR ID can't be known beforehand.